spec/index.bs

```diff
@@ -4428,26 +4435,52 @@ typedef double GPUPipelineConstantValue; // May represent WGSL's bool, f32, i32,
                 : {{GPUBindGroupLayoutEntry/sampler}}
                 :: If |entry|.{{GPUBindGroupLayoutEntry/sampler}}.{{GPUSamplerBindingLayout/type}} is:
                     <dl class=switch>
-                        : {{GPUSamplerBindingType/"filtering"}}
-                        :: the |binding| is a non-comparison sampler
-                        : {{GPUSamplerBindingType/"non-filtering"}}
-                        :: the |binding| is a non-comparison sampler
+                        : {{GPUSamplerBindingType/"filtering"}} or {{GPUSamplerBindingType/"non-filtering"}}
+                        :: |binding| has type `sampler`.
                         : {{GPUSamplerBindingType/"comparison"}}
-                        :: the |binding| is a comparison sampler
+                        :: |binding| has type `comparison_sampler`.
                     </dl>
 
                 : {{GPUBindGroupLayoutEntry/texture}}
-                :: If |entry|.{{GPUBindGroupLayoutEntry/texture}}.{{GPUTextureBindingLayout/multisampled}} is:
+                :: If, and only if,
+                    |entry|.{{GPUBindGroupLayoutEntry/texture}}.{{GPUTextureBindingLayout/multisampled}}
+                    is `true`, |binding| has type `texture_multisampled_2d<T>`.
```

Comment:
this will need to be adjusted when #1793 lands

Reply:
thanks for remembering this, I expected to have to fix this before landing, but forgot to add a note